lib/ui/login/login_screen.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class LoginScreen extends StatelessWidget {
+  const LoginScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    SystemChrome.setSystemUIOverlayStyle(const SystemUiOverlayStyle(
+        statusBarColor: Colors.transparent, statusBarIconBrightness: Brightness.light, statusBarBrightness: Brightness.dark));
+    return Scaffold(
+        backgroundColor: Colors.transparent,
+        body: Container(
+          decoration: BoxDecoration(
+            image: DecorationImage(
+              image: AssetImage(Assets.images.bgLoginOverlay.path),
+              fit: BoxFit.cover,
+            ),
+          ),
+          child: Padding(
+            padding: const EdgeInsets.symmetric(horizontal: Dimensions.paddingLarge),
+            child: Center(
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Padding(
+                      padding: const EdgeInsets.only(bottom: 109.0, left: 104.0, right: 104.0),
+                      child: Image.asset(
+                        Assets.images.icNimbleWhiteLogo.path,
+                      )),
+                  Padding(
+                      padding: const EdgeInsets.only(bottom: Dimensions.paddingMedium),
+                      child: InputFieldWidget(
+                        key: const Key('login_email'),
+                        textHint: AppLocalizations.of(context)!.loginEmail,
+                        keyboardType: TextInputType.emailAddress,
+                        textInputAction: TextInputAction.next,
+                      )),
+                  Padding(
+                      padding: const EdgeInsets.only(bottom: Dimensions.paddingMedium),
+                      child: InputFieldWidget(
+                          key: const Key('login_password'),
+                          textHint: AppLocalizations.of(context)!.loginPassword,
+                          keyboardType: TextInputType.visiblePassword,
+                          isPassword: true,
+                          textSuffixButton: AppLocalizations.of(context)!.loginForgotPassword,
+                          textSuffixButtonCallback: null)),
+                  ElevatedButton(
+                    onPressed: () {},
+                    style: ElevatedButton.styleFrom(
+                      minimumSize: const Size.fromHeight(Dimensions.buttonHeight),
+                      backgroundColor: Colors.white,
+                      shape: RoundedRectangleBorder(
+                        borderRadius: BorderRadius.circular(Dimensions.radiusNormal),
+                      ),
+                    ),
+                    child: Text(
+                      AppLocalizations.of(context)!.login,
+                      style: Theme.of(context).textTheme.titleSmall?.copyWith(color: Colors.black),
+                    ),
+                  ),
```

Comment:
To make the code readable and maintainable, please consider splitting it into small chunks, like variables and functions.

Besides, as you're using Column, you can also use `SizedBox` for setting the vertical spacing, instead of wrapping widgets inside `Padding`. 

```suggestion
                  _nimbleLogo,
                  const SizedBox(height: 109),
                  _emailInput,
                  const SizedBox(height: Dimensions.paddingMedium),
                  _passwordInput,
                  const SizedBox(height: Dimensions.paddingMedium),
                  _loginButton,
```